FND/Accounting/AccountingEquity.rdf

```diff
@@ -102,6 +102,21 @@
 		<skos:definition>asset in the form of stocks, bonds, rights, certificates, bank balances, etc., as distinguished from tangible, physical assets or intangible assets such as intellectual property</skos:definition>
 	</owl:Class>
 	
+	<owl:Class rdf:about="&fibo-fnd-acc-aeq;Income">
+		<rdfs:subClassOf>
+			<owl:Class>
+				<owl:unionOf rdf:parseType="Collection">
+					<rdf:Description rdf:about="&fibo-fnd-acc-cur;AmountOfMoney">
+					</rdf:Description>
+					<rdf:Description rdf:about="&fibo-fnd-acc-aeq;FinancialAsset">
+					</rdf:Description>
+				</owl:unionOf>
+			</owl:Class>
+		</rdfs:subClassOf>
+		<rdfs:label xml:lang="en">income</rdfs:label>
+		<skos:definition>cash or cash equivalent(s) received during a period of time in exchange for labor or services, from the sale of goods or property, or as profit from financial investments</skos:definition>
```

Comment:
There is no property to represent the period of time for which the Income applies.

Reply:
Ok, done.